docs/schedule/examples/flex.md

```diff
@@ -0,0 +1,326 @@
+
+### Define trips
+
+Route 476 offers on-demand services between each stop in the Angermünde region, including two trips(services): one for weekdays and the other for weekends. 
+
+[**trips.txt**](../../reference/#tripstxt)
+
+route_id | service_id | trip_id 
+-- | -- | -- 
+476 | on_demand_weekdays | 476_weekdays 
+476 | on_demand_weekends | 476_weekends 
+
+### Define location groups
+
+As riders can book services between each stop, to avoid defining all stop-to-stop combinations in stop_times.txt, the appropriate approach is to define these stops as a location group using location_groups.txt and location_group_stops.txt.
+
+[**location_groups.txt**](../../reference/#location_groupstxt)
+
+location_group_id | location_group_name 
+-- | -- 
+476_stops | durch den RufBus 476 bedientes Gebiet im Raum Angermünde
+
+[**location_group_stops.txt**](../../reference/#location_group_stopstxt)
+
+location_group_id | stop_id 
+-- | -- 
+476_stops | de:12073:900340004::1
+476_stops | de:12073:900340004::2
+476_stops | de:12073:900340004::3
+476_stops | de:12073:900340004::4
+476_stops | de:12073:900340100::1
+476_stops | de:12073:900340100::2
+476_stops | ...
+
+### Define booking rules
+
+- The 476 route service requires booking at least one hour in advance. Using `booking_type = 1` indicates that the service requires up to same-day booking with advance notice. The `prior_notice_duration_min = 60` indicates a requirement for booking at least 60 minutes in advance.
+- There are slight differences between weekday and weekend bookings, so separate booking rules can be defined for weekday and holiday services. More details can be provided in the `message` field.
+- Information and booking page links can be provided in the `info_url` and `booking_url` fields.
+
+[**booking_rules.txt**](../../reference/#booking_rulestxt)
+
+booking_rule_id | booking_type | prior_notice_duration_min | message | phone_number | info_url | booking_url
+-- | -- | -- | -- | -- | -- | --
+flächenrufbus_angermünde_weekdays | 1 | 60 | Anmeldung mind. 60min vorher erforderlich, per Anruf zwischen 08:00 und 24:00 möglich, oder online rund um die Uhr | +49 3332 442 755 | https://uvg-online.com/rufbus-angermuende/ | https://uvg.tdimo.net/bapp/#/astBuchungenView
+flächenrufbus_angermünde_weekends | 1 | 60 | 1€ Komfortzuschlag pro Person; Anmeldung mind. 60min vorher erforderlich, per Anruf zwischen 08:00 und 24:00 möglich, oder online rund um die Uhr | +49 3332 442 755 | https://uvg-online.com/rufbus-angermuende/ | https://uvg.tdimo.net/bapp/#/astBuchungenView
+
+### Define stop_times
+
+- The 476 route operates from 5:30 PM to 10:00 PM on weekdays and from 8:00 AM to 10:00 PM on weekends. The operating hours are defined using the `start_pickup_drop_off_window` and `end_pickup_drop_off_window` fields.
```

Comment:
I'd remove the first level of bullet points

Reply:
Updated format